src/test/training/supportbank/TransactionTest.java

```diff
@@ -0,0 +1,30 @@
+package training.supportbank;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TransactionTest {
+
+    Transaction valid = new Transaction("22/07/2017", "personFrom", "personTo", "narrative", "21.70");
+
+    @Test
+    public void isValidReturnsFalseWhenAmountInvalid() {

```

Comment:
Minor style point - test names like "fooReturnsFalse" can often be reworded to describe the behaviour more clearly, without using code-specific language like "returns". Maybe something like "isInvalidWhenAmountIsNotANumber"?